### ISSUE TITLE
Navigation bar buttons are hardly visible on some devices

### ISSUE THREAD

**Issue by TitanNano** [mozilla-mobile/fenix#519]
## Steps to reproduce
Open Fenix

### Expected behavior
Navigation bar buttons should be clearly visible.

### Actual behavior
I'm on android 7 and it's pretty hard to see the navigation bar buttons in the default Fenix theme. They only have a slight black drop shadow, but the icons them self remain white.

![photo5417842305076931232](https://user-images.githubusercontent.com/4500858/52842851-a5077080-3100-11e9-8e26-72acc1fe4edf.jpg)

### Device information

* Android device: 7.1.1
* Fenix version: 1.0.1907

**Comment by sblatz** [comment]
This is likely due to the fact that we can't set `windowLightNavigationBar` on <v27. We should consider using a different color nav bar for these versions.

**Comment by brampitoyo** [comment]
Let me take this issue on. Thanks @TitanNano for catching this issue!

@sblatz I will ask the team about the right dark colour to use, and get back to you.

**Comment by bsurd** [comment]
This is also reproducible on an HTC Desire 820 (Android 6.0.1)

The issue is not reproducible when switching to Private Mode since the buttons remain white.

![normal](https://user-images.githubusercontent.com/17178945/53885552-b7513c00-4026-11e9-9a6e-53cbcb60228c.png)
![private](https://user-images.githubusercontent.com/17178945/53885500-8ffa6f00-4026-11e9-86e0-5fb10e6c8553.png)

**Comment by vesta0** [comment]
@eeejay let us know if you have any recommendations.

**Comment by eeejay** [comment]
I don't have any further recommendations here. Looks like there is an obvious fix.

**Comment by ekager** [comment]
Closing for #1144
